cmd/istioctl/main.go

```diff
@@ -288,6 +303,10 @@ func init() {
 	}
 	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", defaultKubeconfig,
 		"Kubernetes configuration file")
+
+	rootCmd.PersistentFlags().StringVarP(&istioNamespace, "istioNamespace", "ns", kube.IstioNamespace,
```

Comment:
ins or in ? (to be consistent with -n being the app namespace)

Reply:
changed to -i.

Reply:
I suggest not to use short flags at all, except when it's obvious what they mean.

Reply:
+1. This is not intuitive. --istio-namespace is better.